sdk/storage/azure_storage_blob/src/parsers.rs

```diff
@@ -37,3 +37,26 @@ pub fn format_page_range(offset: u64, length: u64) -> Result<String, Error> {
     let content_range = format!("bytes={}-{}", offset, end_range);
     Ok(content_range)
 }
+
+/// Takes in a HashMap of tag key-value pairs and converts them to a filter expression
+/// for use with [`BlobServiceClient::find_blobs_by_tags()`](crate::BlobServiceClient::find_blobs_by_tags) or [`BlobContainerClient::find_blobs_by_tags()`](crate::BlobContainerClient::find_blobs_by_tags).
+///
+/// # Arguments
+///
+/// * `tags` - A HashMap containing tag key-value pairs representing the
+///   expression to find blobs whose tags match the specified condition.
+pub fn format_filter_expression(tags: &HashMap<String, String>) -> Result<String, Error> {
+    if tags.is_empty() {
+        return Err(Error::new(
+            ErrorKind::InvalidInput,
+            "Tags HashMap cannot be empty.".to_string(),
+        ));
+    }
+
+    let format_expression: Vec<String> = tags
+        .iter()
+        .map(|(key, value)| format!("\"{}\"='{}'", key, value))
```

Comment:
Double quotes for keys and single quotes for values? You can't just use single quotes for both? I mean, it's not a problem - just stood out as odd.